cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -88,8 +86,8 @@ func pullRepositories(repositories []abaputils.Repository, pullConnectionDetails
 		if err != nil {
 			break
 		}
-		finishPullLogs()
 	}
+	finishPullLogs()
```

Comment:
Is this the right spot for finishPullLogs()?
It seems that this will be also printed in case of an error

Reply:
It must be the appropriate spot 🙈 Inside the loop you you have a logger in the `handlePull` method which logs the if a SINGLE repository has been pulled successfully.
If there is any error related to the pull over all repos --> `err` obj will break the loop. Hence, this will not be printed in case of an error and only if every software component gets pulled successfully!